app/models/conference.rb

```diff
@@ -10,15 +10,14 @@ class Conference < ActiveRecord::Base
 
   include HasSeason
 
-  has_many :conference_preferences, dependent: :destroy
-  has_many :attendees, through: :conference_preferences, source: :team
+  has_many :preferences_first_conferences, :class_name => 'ConferencePreference', :foreign_key => 'first_conference_id'
```

Comment:
Comes down to personal preference, but I'd say a naming like:
```ruby
has_many :first_choice_conference_preferences
has_many :second_choice_conference_preferences
```
could be a bit more readable. What do you think?